index.html

```diff
@@ -8,7 +8,180 @@
     <title>Africa AI summit 2023</title>
   </head>
   <body>
-    <main></main>
+    <header>
+      <nav>
+        <button class="barbutton" id="barbutton">
+          <i class="fa-sharp fa-solid fa-bars fa-2xl" id></i>
+        </button>
+        <ul class="linkto" hidden>
+          <li>
+            <a href="https://github.com/zewdu444/" target="_blank"
+              ><i class="fa-brands fa-github"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/zewdu444" target="_blank"
+              ><i class="fa-brands fa-twitter"></i
+            ></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+        <div class="navwrapper" hidden>
+          <a href="index.html" class="logolink"
+            ><img
+              class="headerlogo"
+              src="./images/ailogo.png"
+              alt="Africa AI conference"
+          /></a>
+          <ul class="navlist" hidden>
+            <li class="about"><a href="about.html">About</a></li>
+            <li><a href="#programs">Program</a></li>
+            <li><a href="#speakers">Speaker</a></li>
+            <li><a href="#partners">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+            <li class="aievent"><a href="index.html">AIevent</a></li>
+          </ul>
+        </div>
+
+        <!-- mobile popup menu -->
+        <div class="popupmenu" id="mobileoption" hidden>
+          <button class="closebutton" id="closebutton">
+            <i class="fa-2xl fa-solid fa-xmark"></i>
+          </button>
+          <ul class="popnavlist">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#programs">Program</a></li>
+            <li><a href="#speakers">Speaker</a></li>
+            <li><a href="about.html#partners">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+            <li><a href="index.html">AIevent</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="maincontainer">
+      <section class="eventsection">
+        <!-- event description  -->
+        <h2 class="subtitle">Empowering Africa with AI</h2>
+        <h1 class="title">
+          AFRICAN CONFERENCE ON ARTIFICIAL INTELLIGENCE 2023
+        </h1>
```

Comment:
- For semantic reasons, it is important that your `hx` tags follow an ascending hierarchy. i.e, an `h2` tag should be used after an `h1` tag, not vice-versa. PLease fix this 🙏

Reply:
- [ ] @mckent05 `hx` elements place in ascending order